Packs/SkyboxSecurity/Integrations/SkyboxTicketsService/SkyboxTicketsService.py

```diff
@@ -0,0 +1,2281 @@
+            static_init_db = os.getenv('ZEEP_STATIC_CACHE_DB', '/zeep/static/cache.db')
+            if os.path.isfile(static_init_db):
+                demisto.debug(f'copying static init db: {static_init_db} to: {db_path}')
+                shutil.copyfile(static_init_db, db_path)
+    except Exception as ex:
+        # non fatal
+        demisto.error(f'Failed copying static init db to: {db_path}. Error: {ex}')
+    return db_path
+
+
+def getTicketWorkflow_command(client, args):
+    ticketId = args.get('ticketId')
+
+    response = client.service.getTicketWorkflow(ticketId)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketWorkflow',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getOriginalChangeRequestRouteInfoV1_command(client, args):
+    ticketId = args.get('ticketId')
+    changeRequestId = args.get('changeRequestId')
+
+    response = client.service.getOriginalChangeRequestRouteInfoV1(ticketId, changeRequestId)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getOriginalChangeRequestRouteInfoV1',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getTicketTypePhasesByTicketType_command(client, args):
+    ticketType = args.get('ticketType')
+
+    response = client.service.getTicketTypePhasesByTicketType(ticketType)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketTypePhasesByTicketType',
+        outputs_key_field='id',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getOriginalChangeRequestV7_command(client, args):
+    ticketId = args.get('ticketId')
+
+    response = client.service.getOriginalChangeRequestV7(ticketId)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getOriginalChangeRequestV7',
+        outputs_key_field='',
+        readable_output=tableToMarkdown("Original Change Request", serialize_object_list(response)),
+        outputs=serialize_object_list(response),
+        raw_response=serialize_object_list(response)
+    )
+
+    return command_results
+
+
+def setTicketFields_command(client, args):
+    ticketId = args.get('ticketId')
+    ticketIdType = args.get('ticketIdType')
+    ticketFields_typeCode = args.get('ticketFields_typeCode')
+    ticketFields_value = args.get('ticketFields_value')
+
+    ticketFields_type = client.get_type('ns0:ticketField')
+    ticketFields = ticketFields_type(
+        typeCode=ticketFields_typeCode,
+        value=ticketFields_value,
+    )
+
+    response = client.service.setTicketFields(ticketId, ticketIdType, [ticketFields])
+    command_results = CommandResults(
+        outputs_prefix='Skybox.setTicketFields',
+        outputs_key_field='',
```

Comment:
```suggestion
        outputs_key_field='ticket_id',
```

Reply:
Fixed as part of commit `7290d4d92a4d55337df8fd68a160b9912e8df459`